LogMonitor/src/LogMonitor/ConfigFileParser.cpp

```diff
@@ -593,6 +595,7 @@ void _PrintSettings(_Out_ LoggerSettings& Config)
             std::wprintf(L"\t\tDirectory: %ls\n", sourceFile->Directory.c_str());
             std::wprintf(L"\t\tFilter: %ls\n", sourceFile->Filter.c_str());
             std::wprintf(L"\t\tIncludeSubdirectories: %ls\n", sourceFile->IncludeSubdirectories ? L"true" : L"false");
+            std::wprintf(L"\t\includeFileNames: %ls\n", sourceFile->IncludeFileNames ? L"true" : L"false");

```

Comment:
Seems like we let a small typo in.

(L"\t\includeFileNames  should be (L"\t\tIncludeFileNames. There is a missing \t then we have IncludeFileNames.


std::wprintf(L"\t\tIncludeFileNames: %ls\n", sourceFile->IncludeFileNames ? L"true" : L"false");